apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx

```diff
@@ -15,18 +15,24 @@ thumb: launch-week-6/pggraphql/og-pg-graphql.png
 ---
 
 It's been 4 months since the [1.0.0 release of pg_graphql](https://supabase.com/blog/pg-graphql-v1).
-Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces. 
+Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces.
 
 In this article, we’ll walk through those features and show examples of each.
 
+<div className="bg-gray-300 rounded-lg px-6 py-2 italic">
+
+📢 These features are only available after upgrading your project to v1.2.0. For help with upgrading, please review the documentation [here](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects).
```

Comment:
this probably need to be more clear that its referring to "Graphql" versioning. 
It also suggests "upgrading", but then links to Supabase migrating and upgrading.. I assume what it needs to really suggest is upgrading Postgres ?

Reply:
cc @gregnr

Reply:
cc @olirice

Reply:
Good catch @MildTomato.

@olirice I can update this - is this PG version correct: `15.1.0.63`?

https://github.com/supabase/postgres/commit/ccd2e902cf528a59623b2924c12053e5138681ff

Reply:
thanks, yes it is `15.1.0.63`

Reply:
Updated 👍